StRoot/StJetMaker/towers/StjTowerEnergyCutEt.h

```diff
@@ -12,14 +12,20 @@
 class StjTowerEnergyCutEt : public StjTowerEnergyCut {
 
 public:
-  StjTowerEnergyCutEt(double min = 0, double max = 50000.0)
-    : _min(min), _max(max) { }
+  StjTowerEnergyCutEt(double min = 0, double max = 50000.0, bool respect_vertex_position = true)
+    : _min(min), _max(max), _flag(flag) { }
```

Comment:
Well you need to update the member initialization as well...

```suggestion
    : _min(min), _max(max), _flag(respect_vertex_position) { }
```